auditbeat/auditbeat.reference.yml

```diff
@@ -38,6 +38,9 @@ auditbeat.modules:
   rate_limit: 0
   include_raw_message: false
   include_warnings: false
+  reload:
+    enabled: false
+    period: 30s
```

Comment:
The tests are failing because this file is actually generated, the proper file to add these additions to is `auditbeat/module/auditd/_meta/config.yml.tmpl`